tests/integration/components/frost-subcontext-actions-test.js

```diff
@@ -0,0 +1,53 @@
+import {expect} from 'chai'
+import {setupComponentTest} from 'ember-mocha'
```

Comment:
`ember-frost-test` has new utilities for this kind of thing.  No big deal, but worth learning the best practice